nservicebus/azure/publisher-names-configuration.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Publishers name configuration
+summary: Configuration mapping between publisher names and message types for Endpoint Oriented Topology
+tags:
+- Cloud
+- Azure
+- Transports
+---
+
+When Endpoint Oriented Topology is selected, mapping between publisher names and message types has to be properly configured at transport level to ensure that message routing works as expected.  
+Mapping can be configured once message type at time:
```

Comment:
> +Mapping can be configured once message type at time:

Mapping can be configured for a specific message type:

> or supplying an assembly to scan.

or by supplying an assembly to scan